### ISSUE TITLE
Chat Component, Chat.Item a new slot needed

### ISSUE THREAD

**Issue by Octavianus** [microsoft/fluent-ui-react#1303]
### Problem description
New slot needed for read receipts feature, beside any message, there should be a slot at bottom left of the message's right blank area to indicate the read status. 
@alinais 

### Proposed solution
I post the idea looks in MVP section. 
As proposed by @jurokapsiar: I assume a new slot will need to be created for the chat message.

Accessibility: 
1. Mouse: Hover over the slot should have a text popover.
2. Screen reader: The text popover I mentioned above should append to the end of each message to tell the user current message's status.

### MVP
![imgo](https://user-images.githubusercontent.com/2127328/57409966-6a145680-719e-11e9-93ab-e783d39ebb65.jpg)

**Comment by levithomason** [comment]
Yes, read receipts are common enough in all chat applications to warrant a slot for it.  This will also make it targetable for theming.

You can achieve this for now by absolutely positioning an element within the Chat Message content.
